dogpile/cache/api.py

```diff
@@ -548,3 +550,22 @@ def set_serialized_multi(
 
         """
         raise NotImplementedError()
+
+
+@dataclass(frozen=True)
```

Comment:
**Federico Caselli** (CaselIT) wrote:

I think dobpile is still supporting 3.6, no dataclasses there

If like mike suggested this is for 1.3 maybe 3.6 (and 3.7 too) could be both removed. Let's wait for his input on this one

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/5029

Reply:
**Michael Bayer** (zzzeek) wrote:

we should bump to 3.8 as minimum, 3.7 is EOL

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/5029

Reply:
Is there already a PR somewhere to bump requirements to 3.8?
What does it mean in term of release schedule for the current PR? (I need this feature and I'd like to know if I can wait for this change to be merged upstream or if I should "fork" on my side).

Reply:
well we can't release this without bumping the reqs so we'd have to do that also, there's no "schedule" per se, we can get a PR to bump up to 3.8 alone with a changelog note, I will update the CI and that's all we'd need.

there's no reason you can't use dogpile private API for the moment with a version pin, there's nothing else happening with dogpile right now.

Reply:
ok thanks

Reply:
other than changing the `python-requires` in the setup file is there anything else required to support only 3.8+ (and the changelog)

Reply:
no not really anything, we can modernize the python internals subsequent to that bump

Reply:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/5029